src/eval/operation.rs

```diff
@@ -1258,6 +1258,29 @@ impl<R: ImportResolver, C: Cache> VirtualMachine<R, C> {
                     ))
                 }
             },
+            UnaryOp::Trace() => {
+                if let Term::Str(s) = &*t {
+                    println!("builtin.trace: {s}");
+                    Ok(())
+                } else {
```

Comment:
Just using `println!` might not be the best option. E.g. I don't know how that interacts with the WASM REPL.

Reply:
After trying it locally in the WASM playground, it doesn't seem to break anything. But the REPL in the playground just doesn't print out any messages.

Reply:
I think I'll make an issue to track that. For now, I'll just change `println!` to `eprintln!` to make this more compatible with invocations like `nickel export -f test.ncl > test.json`.